bump.sh

```diff
@@ -23,9 +23,6 @@ export NODE_OPTIONS="--max-old-space-size=4096 ${NODE_OPTIONS:-}"
 
 npx lerna version ${version} --yes --exact --force-publish=* --no-git-tag-version --no-push
 
-# Another round of install to fix package-lock.jsons
-/bin/bash ./install.sh
-
```

Comment:
why?

Reply:
There are no `package-lock.json` files to fix anymore, and this step wasn't adding any value - just making things noisier and slower.